### PR TITLE
Generate new compute import for SDK

### PR DESCRIPTION
**Pull request by szimmer-dev**

### CODE REVIEW

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Compute</PackageId>
     <Description>Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
-    <Version>28.0.0</Version>
+    <Version>28.1.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Compute</AssemblyName>
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
```
Comment (hyonholee) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj]: Could you update PackageReleaseNotes?  (Briefly describe this update)

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("28.0.0.0")]
-[assembly: AssemblyFileVersion("28.0.0.0")]
+[assembly: AssemblyVersion("28.1.0.0")]
```
Comment (hyonholee) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs]: For minor version update, you don't have to change AssemblyVersion.  Just update AssemblyFileVersion.

**Review by hyonholee — CHANGES_REQUESTED** [review]

**Comment by erich-wang** [comment]
@szimmerMicrosoft , could you
1. Provide the swagger spec PR related to this PR?
2. Address the comment about AssemblyVersion

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (szimmerMicrosoft) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs]: Fixed

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj (hunk 1 above)
Comment (szimmerMicrosoft) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj]: Updated

**Comment by szimmerMicrosoft** [comment]
> @szimmerMicrosoft , could you
> 
> 1. Provide the swagger spec PR related to this PR?
> 2. Address the comment about AssemblyVersion

 have addressed the assembly version.

Here is the source pull request:
https://github.com/Azure/azure-rest-api-specs/pull/6683

**Review by erich-wang — APPROVED** [review]

**Comment by erich-wang** [comment]
Approve from my side. @hyonholee , please approve if no more comments.

**Review by hyonholee — APPROVED** [review]

**Comment by hyonholee** [comment]
Approved.  BTW, @erich-wang   could you check why CI build failed with the following error? It doesn't seem to be related with this PR.

Failed to resolve SDK 'Microsoft.Build.CentralPackageVersions/2.0.36'. Package restore was successful but a package with the ID of "Microsoft.Build.CentralPackageVersions" was not installed.

**Comment by hyonholee** [comment]
Hi,  @erich-wang could you merge this PR, if there is no more concern?  Thanks.

**Pull request merged**